src/main/java/uk/gov/hmcts/divorce/systemupdate/event/SystemPronounceCase.java

```diff
@@ -102,4 +136,31 @@ private void generateConditionalOrderGrantedDocs(final CaseDetails<CaseData, Sta
         }
     }
 
+    private void updateMissingFields(CaseData caseData, User user, String serviceAuth) {
+
+        try {
+            final BoolQueryBuilder query = boolQuery()
+                    .must(matchQuery("data.bulkListCaseReferenceLink.CaseReference", caseData.getBulkListCaseReferenceLink()
+                            .getCaseReference()))
+                    .must(existsQuery("data.coCourt"));
+
+            Map<String, Object> otherCaseData =
+                    ccdSearchService.searchForCasesWithQuery(0, 1, query, user, serviceAuth)
+                            .getCases().stream().findFirst().orElseThrow().getData();
+
+            CaseData convertedCaseData = objectMapper.convertValue(otherCaseData, CaseData.class);
+
+            ConditionalOrder conditionalOrder = caseData.getConditionalOrder();
+
+            conditionalOrder.setCourt(convertedCaseData.getConditionalOrder().getCourt());
+            conditionalOrder.setDateAndTimeOfHearing(convertedCaseData.getConditionalOrder().getDateAndTimeOfHearing());
+            conditionalOrder.setOfflineCertificateOfEntitlementDocumentSentToApplicant1(NO);
+            conditionalOrder.setOfflineCertificateOfEntitlementDocumentSentToApplicant2(NO);
```

Comment:
I'm thinking we may need to open up permissions for these fields to give CRU access to caseworkers 🤔. ConditionalOrder looks to have default access in CaseData at the moment, which gives the CW Read permissions